scripts/add_reply_url_for_cluster.sh

```
@@ -15,6 +15,7 @@
 #   K8S_NAMESPACE           (Mandatory)
 #   K8S_INGRESS_NAME        (Mandatory)
 #   REPLY_PATH              (Mandatory)
+#   SILENT_MODE             (Optional. Defaulted if omitted. ex: false,true)
```

Can we document what `SILENT_MODE` does? Is this no stdout output, no logs, no notifications…?

---

Agree. Silent does not do what you think it does. It silently accepts the questions to confirm if account is correct and similar. So a better naming would probably be ACCEPT or something. Silent does not filter any output at all.

---

Changed it to USER_PROMPT:

https://github.com/equinor/radix-platform/commit/aea6bd700a83a4b32a4ba39fe1dcc5fb5597e87b